system/screen.go

```diff
@@ -188,14 +188,19 @@ func (sc *Screen) ConstrainWinGeom(sz, pos image.Point) (csz, cpos image.Point)
 		csz.Y = scsz.Y - 10
 		// fmt.Println("constrain y:", csz.Y)
 	}
-
-	// these are windows-specific special numbers for minimized windows
-	// can be sent here for WinGeom saved geom.
-	if cpos.X == -32000 {
-		cpos.X = 0
-	}
-	if cpos.Y == -32000 {
-		cpos.Y = 50
+	if TheApp.Platform() == Windows {
+		if csz.Y > (scsz.Y - 20) { // leave room for title
```

Comment:
There is a programmatic way to determine this with [glfwGetWindowFrameSize](https://www.glfw.org/docs/3.3/window_guide.html#window_size:~:text=The%20above%20functions%20work%20with%20the%20size%20of%20the%20content%20area%2C%20but%20decorated%20windows%20typically%20have%20title%20bars%20and%20window%20frames%20around%20this%20rectangle.%20You%20can%20retrieve%20the%20extents%20of%20these%20with%20glfwGetWindowFrameSize.).

Reply:
this has been implemented.